src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -159,7 +159,25 @@ public class StripePaymentService : IPaymentService
                     InvoiceSettings = new CustomerInvoiceSettingsOptions
                     {
                         DefaultPaymentMethod = stipeCustomerPaymentMethodId
-                    }
+                    },
+                    Address = new AddressOptions
+                    {
+                        Country = taxInfo.BillingAddressCountry,
+                        PostalCode = taxInfo.BillingAddressPostalCode,
+                        // Line1 is required in Stripe's API, suggestion in Docs is to use Business Name intead.
+                        Line1 = taxInfo.BillingAddressLine1 ?? org.BusinessName,
```

Comment:
does it work if empty/whitespace though?

Reply:
I'll give it a go.

Reply:
That worked! Passed empty string via the API, accepted and it stores `null` in Stripe for Address Line 1.